public/script.js

```diff
@@ -0,0 +1,55 @@
+//access elements from html
+const inputSearch = document.querySelector('.input-search')
+const buttSearch = document.querySelector('.btn-search')
+buttSearch.addEventListener('click', () => {
+    const data = inputSearch.value;
+    const options = {
+        method: 'POST',
```

Comment:
Here, you are not creating or modifying any resource nor sending sensitive data, you are just retrieving(fetching) data from some resources, the method used here is supposed to be GET not POST. And you can send the along the search value using query string `fetch('/search?q=${input.value}')`

[GET vs POST](https://stackoverflow.com/questions/3477333/what-is-the-difference-between-post-and-get)